readme.md

```diff
@@ -155,6 +156,8 @@ All about development on EOSIO platform.
 - [Transit API for EOS](https://www.eostransit.com/) - Connecting signature providers and dApps through an open-source, modular, and simple standard for EOSIO-based blockchains.
   - [eosnewyork/eos-transit](https://github.com/eosnewyork/eos-transit) - Official GitHub.
   - [The Transit API: Connecting dApps & Signature Providers](https://medium.com/eos-new-york/the-transit-api-connecting-dapps-signature-providers-5d816c056f7f) - Article.
+- [@greymass/eosio](https://github.com/greymass/eosio-core/) - Strongly typed JavaScript library
```

Comment:
Could you have a look at CI logs https://github.com/DanailMinchev/awesome-eosio/runs/2687586278 , awesome-lint requires all of the descriptions to end with `.` and could you make use `greymass` instead of `@greymass` for consistency